File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int n; //variable for size
    do
    {
        n = get_int("Size(1-8): "); //prompt user for size
    }
    while (n > 8 || n < 1); //keeps prompting user when n is not 1-8

    for (int i = 0; i < n; i++) //for each row
    {
        for (int j = 1; j < n - i; j++) //blanks on left side for each row
        {
            printf(" ");
        }

        for (int k = 0; k < i + 1; k++) //hashes on left side
        {

            printf("#");
        }

        printf("  "); //two blanks in the middle

        for (int l = 0; l < i + 1; l++) //hashes on right side
        {
            printf("#");
        }

        printf("\n"); //move to next row

    }
    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    size = 0
    # keeps promting user for size 1-8
    while size > 8 or size < 1:
        size = get_int("Size(1-8): ")
    # for each row
    for i in range(size):
        # blanks on left side
        for j in range(1, size - i):
            print(" ", end="")
        # hashes on left side
        for k in range(i + 1):
            print("#", end="")
        # two blanks in middle
        print("  ", end="")
        # hashes on right side
        for l in range(i + 1):
            print("#", end="")
        print("")


if __name__ == "__main__":
    main()